tests/descriptor_tests/test_git.py

```diff
@@ -208,3 +212,84 @@ def test_branch(self):
         latest_desc.copy(copy_target)
         self.assertTrue(os.path.exists(os.path.join(copy_target, ".git")))
 
+    @skip_if_git_missing
+    def test_downloads_to_bundle_cache(self):
+        """
+        Tests local downloads to the bundle cache for git descriptors.
+        """
+
+        # skip this test on windows or py2.5 where multiprocessing isn't available
+        if sys.platform == "win32" or sys.version_info < (2, 6):
+            return
+
+        def _download_bundles(target=None):
+            """
+            :param target: The path to which the bundle is to be downloaded.
```

Comment:
Oh, I see now!!! Is it not at the class level because it's being invoked by the multiprocessing module?

Reply:
I added it to the test method as it wasn't being invoked by other test functions. As you mentioned the attempt to avoid nested functions, it could be moved to the class as well.